backend/requirements.txt

```diff
@@ -1,5 +1,4 @@
 Django
-psycopg2
```

Comment:
this is still a production requirement, so I think we should probably make a requirements-prod file or something if you wanna remove it from here